eslint/README.md

```diff
@@ -0,0 +1,11 @@
+## ESLint
```

Comment:
Let's write few sentences on how to use those configs and what - if any - packages should be installed prior using it.

Reply:
Done